src/FissionErrors.ts

```diff
@@ -0,0 +1,73 @@
+export type ByteLength = number
+
+export interface AxiosError extends Error {
+    isAxiosError: boolean
```

Comment:
Given that this interface is `AxiosError` should it always be an axios error? We may not want to keep this field